packages/app-mobile/components/voiceTyping/AudioRecordingBanner.tsx

```
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { PrimaryButton, SecondaryButton } from '../buttons';
 import { _ } from '@joplin/lib/locale';
 import { useCallback, useEffect, useRef, useState } from 'react';
-import { Audio, InterruptionModeIOS } from 'expo-av';
+import { useAudioRecorder as useExpoAudioRecorder, AudioModule, RecordingPresets, RecordingOptions, AndroidOutputFormat, AndroidAudioEncoder, IOSAudioQuality, IOSOutputFormat } from 'expo-audio';
```

ESLint

[error] 'RecordingOptions' is defined but never used.

(no-unused-vars)